filament/backend/src/vulkan/memory/ResourcePointer.h

```diff
@@ -156,6 +162,7 @@ struct resource_ptr {
     // only be used from VulkanDriver.
     inline void dec() {
         assert_invariant(mRef);
+        mRef->setHandleConsiderDestroyed();
```

Comment:
should we at least `assert_invariant()` here that the handle is not already in the considered destroyed state? (I think this should never happen right?).

Reply:
done